src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+table.tHead.addEventListener('click', (happening) => {
```

Comment:
```suggestion
table.tHead.addEventListener('click', (event) => {
```
event was okay, I just didn't want you to shorten it